src/pal/inc/unixasmmacrosarm.inc

```diff
@@ -147,11 +147,15 @@ C_FUNC(\Name\()_End):
         __PWTB_StackAlloc = __PWTB_TransitionBlock
 
         .ifnc \pushArgRegs, DoNotPushArgRegs
-                PUSH_ARGUMENT_REGISTERS
+                push {r0 - r3}
+                .save {r0 - r3}
```

Comment:
I'm now testing if .save {r0-r3} is required or not. Depending on the current test, this patch might be reduced into just one line. :)